### ISSUE TITLE
Add Wrapped MATIC to WETH9 addresses

### ISSUE THREAD

**Issue by georgeroman** [Uniswap/sdk-core#39]
This will result in consistent handling of the wrapped native currency on Polygon which uses MATIC instead of ETH:
https://github.com/Uniswap/sdk-core/blob/be6b14fa6f9223e0483389d792a9e07eab557a27/src/entities/weth9.ts#L6

**Comment by Florian-S-A-W** [comment]
@georgeroman I agree that this makes sense. I created a pull request to resolve this.

**Comment by ashutosh887** [comment]
I would like to work on this @Florian-S-A-W @georgeroman
